src/client/component/network.cpp

```diff
@@ -104,16 +104,13 @@ namespace network
 			game::NET_OutOfBandPrint(game::NS_SERVER, from, "error\nYou are already connected to the server.");
 		}
 
+
+		//Sys_StringToSockAddr @ 0x59E810 has essentially the same functionality as this function, 
+		//but breaks with multiple clients from the same IP address. Thus, we use this instead.
```

Comment:
create_socket actually has the same functionality 5BD210, which calls 59E810 so im not sure why you removed that code

Reply:
My understanding is that create_socket either called the function to create a socket provided by MWR if the client was not localhost, or performed similar steps to create a socket itself if not, as the function included at 0x59e810 did not work with clients connected to the localhost.

Unfortunately, the function at 0x59e810 does not allow multiple sockets to the same IP address differentiated by port, so electing to only use `create_socket` in place of 0x59e810 is an easy solution.

Reply:
0x59e810 simply converts a string to a sockaddr, as the name suggests.
the function the game uses to create sockets (0x5BD210) didn't seem to work properly when connecting to addresses that weren't localhost, so it was rewritten (create_socket) and that seemed to fix it.

Reply:
This still works after removing the change. I'll revert this.